include/outputs

```diff
@@ -104,15 +104,29 @@ textInfo(){
 }
 
 textFail(){
-  FAIL_COUNTER=$((FAIL_COUNTER+1))
-  EXITCODE=3
+  ## ignore whitelists for current check
+  level="FAIL"
+  for i in $IGNORES; do
+    ignore_value="${i#*${CHECK_NAME}:}"
+    if [[ $1 =~ ${ignore_value} ]]; then
+      level="WARNING"
```

Comment:
```file include/outputs: I need a bit more information to understand from line 107 to 129. Why the WARN and the change on line 129.```
@toniblyx ok let me describe what I was thinking here:

- Added the "WARNING" level so that even if people ignore things in a whitelist, it's still there in the output as a reminder that they might want to fix it later if it's a temporary suppression.
- The loop at line 109 is basically parsing the whitelist line-by-line, splitting the string upto the first occurrence of `$CHECK_NAME:`  so using: ```${var#*SubStr}  # will drop begin of string up to first occur of `SubStr` ```
  - and then checking if that whitelisted item matches the current item raising the failure
  - I could've made a map for this but it's only supported on bash 4 onwards so this keeps it backwards-compat (With some perf cost for the lookup)
 - It we find a match, treat it as a WARNING (And line 118-121 ensures we don't decrement FAIL_COUNTER & EXIT_CODE if we only saw warnings so far)

Line 129 is using the fact that `${level}` already holds FAIL/WARNING as the level so we can just use that to print the right warning